### PR TITLE
Add rever for changelog and backfill news

### LINKED ISSUE #63: Add release notes/changelog

### PR DESCRIPTION
**Pull request by jthorton**
Fixes #63 by using rever to generate the changelog

### CODE REVIEW

**Review by IAlibay — APPROVED** [review]
lgtm thanks!

File: news/backfill.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+**Added:**
+
+* use rever to manage changelog `PR#77 <https://github.com/OpenFreeEnergy/kartograf/pull/77>`_
```
[the remark this answers is not in the document]
Comment (jthorton) [reply news/backfill.rst:3]: Thanks. I wasn't sure what to include, so I went with everything, but user-facing only makes sense!

File: news/backfill.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
+**Added:**
+
+* Added changelog `PR#77 <https://github.com/OpenFreeEnergy/kartograf/pull/77>`_
+* Support mapping multi chain components `PR#47 <https://github.com/OpenFreeEnergy/kartograf/pull/47>`_
+* Option to not break fused rings when creating mappings `PR#56 <https://github.com/OpenFreeEnergy/kartograf/pull/54>`_
+* Added a citation.cff `PR#45 <https://github.com/OpenFreeEnergy/kartograf/pull/45>`_
+
+**Changed:**
+
+* Update CI `PR#43 <https://github.com/OpenFreeEnergy/kartograf/pull/43>`_
```
Comment (IAlibay) [inline news/backfill.rst:10]: This one can be left out too!

**Review by IAlibay — APPROVED** [review]
lgtm

**Pull request merged**